core/XH.js

```diff
@@ -336,10 +336,11 @@ class XHClass {
      *
      * @param {Object} config - options for toast instance.
      * @param {string} config.message - the message to show in the toast.
-     * @param {element} [config.icon] - icon to be displayed
+     * @param {Element} [config.icon] - icon to be displayed
      * @param {number} [config.timeout] - time in milliseconds to display the toast.
      * @param {string} [config.intent] - The Blueprint intent (desktop only)
      * @param {Object} [config.position] - Position in viewport to display toast. See Blueprint Position enum (desktop only).
+     * @param {Element} [config.containerRef] - Parent component ref (defaults to document.body if undefined)
```

Comment:
The icon "{Element}" is actually a react element.

The containerRef "{Element}" is actually an HtmlElement in the blueprint API, but a Component in our API.    We need to be clearer on this.  If we want it to be either,  let's say that and support that.  Otherwise, let just say its a Component defaulting to `null`...  
 

 const containerElement = document.createElement("div");
        container.appendChild(containerElement);